Reference/Security/Two-factor-authentication/index.md

```diff
@@ -83,7 +83,7 @@ namespace Umbraco9
             var member = _memberService.GetByKey(userOrMemberKey);
 
             var twoFactorAuthenticator = new TwoFactorAuthenticator();
-            SetupCode setupInfo = twoFactorAuthenticator.GenerateSetupCode("My application name", member.Username, secret, false);
+            SetupCode setupInfo = twoFactorAuthenticator.GenerateSetupCode("My application name", secret, 200,200  );
```

Comment:
What is the reason for removing the "member.Username"?
What does "200,200" mean? Would be great with a code-comment here, to explain the numbers 💪 

Also, if `member` should not be used, can line 83 be omitted? 🤔

Reply:
Hi, I use another library. In your example, you use this https://github.com/BrandonPotter/GoogleAuthenticator. I see that the version 3 support .NET Core 6.
Sorry for my mistake. You can reject the changes.

Reply:
Ah, I see!
So just to make sure I understand completely - is it the entire PR is can disregard, or only this part? 🤔